aks/windows/mysql_create.cmd

```diff
@@ -0,0 +1 @@
+az mysql flexible-server create  --resource-group resilientSixGroup  --name resilientsix-mysql-db  --database-name cache  --admin-user admin_user --admin-password admin_password --location polandcentral 	--tier GeneralPurpose 	--sku-name Standard_D2ds_v4 	--version 8.0.21   --yes
```

Comment:
we need to create `devices`, `geolocation` and `services databases. Not sure whether we need the `cache` DB here

Reply:
i'm not sure what you mean. Cache db?